bedrock/privacy/templates/privacy/archive/firefox-cliqz-2018-06.html

```diff
@@ -141,7 +141,7 @@ <h3>Measure and support our marketing</h3>
         <li id="referraltracking"><strong>Campaign and Referral Data</strong>: This helps Mozilla understand the effectiveness of our marketing campaigns.</li>
       </ul>
       <p><em>On Desktop</em>: Firefox by default sends Mozilla HTTP data that may be included with Firefox’s installer.  This enables us to determine the website domain or advertising campaign (if any) that referred you to our download page. Read the <a href="https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/telemetry/data/environment.html#attribution">documentation</a> or <a href="https://support.mozilla.org/kb/desktop-privacy">opt-out</a> before installation.</p>
-      <p id="thirdparty"><em>On iOS and Android</em>: Firefox by default sends mobile campaign data to Adjust, our analytics vendor, which has its own <a href="https://www.adjust.com/privacy_policy/">privacy policy</a>.  Mobile campaign data includes a Google advertising ID, IP address, timestamp, country, language/locale, operating system, and app version. Read the <a href="https://firefox-source-docs.mozilla.org/mobile/android/fennec/adjust.html">documentation</a>.</p>
+      <p id="thirdparty"><em>On iOS and Android</em>: Firefox by default sends mobile campaign data to Adjust, our analytics vendor, which has its own <a href="https://www.adjust.com/terms/privacy_policy/">privacy policy</a>.  Mobile campaign data includes a Google advertising ID, IP address, timestamp, country, language/locale, operating system, and app version. Read the <a href="https://firefox-source-docs.mozilla.org/mobile/android/fennec/adjust.html">documentation</a>.</p>
```

Comment:
It looks like they use a dash instead of an underscore here.

```suggestion
      <p id="thirdparty"><em>On iOS and Android</em>: Firefox by default sends mobile campaign data to Adjust, our analytics vendor, which has its own <a href="https://www.adjust.com/terms/privacy-policy/">privacy policy</a>.  Mobile campaign data includes a Google advertising ID, IP address, timestamp, country, language/locale, operating system, and app version. Read the <a href="https://firefox-source-docs.mozilla.org/mobile/android/fennec/adjust.html">documentation</a>.</p>
```